subsystems/LEDSubsystem.java

```diff
@@ -115,6 +121,7 @@ public void solidrgb(LEDColor color){
         for (var i = startIndex; i < m_ledBuffer.getLength(); i++) {
             m_ledBuffer.setRGB(i,color.x, color.y, color.z );
         }
+        changed=true;
```

Comment:
One comment here but applies in general: I don't think this change really works. If the caller just calls solidrg(LEDColor.Yellow) over and over (which is what's happening), then we're going to treat that as a change. I think you probably want to have something that represents the current state of the LEDs and then compare at the top of the method. If the requested state is different than current state then go ahead and do all the setRGB, update current state, and set changed = true or whatever. Otherwise, just return.